CHANGELOG.rst

```diff
@@ -4,6 +4,31 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+v10.0.0
+=======
+
+Release Summary
+---------------
+
+In this major release, the ``connection/aws_ssm`` connection plugin has been fully migrated to the ``amazon.aws`` collection (https://forum.ansible.com/t/aws-ssm-connection-refactoring-plugin-promotion/39930). Playbooks or Inventory using the Fully Qualified Collection Name (FQCN) for this connection plugin should be updated to use ``amazon.aws.aws_ssm``.
```

Comment:
Can you also please mention boto3 and botocore versions update and python 3.8 removal?